pilot/pkg/xds/bench_test.go

```diff
@@ -250,6 +254,43 @@ func BenchmarkSecretGeneration(b *testing.B) {
 	runBenchmark(b, v3.SecretType, secretCases)
 }
 
+var wdsCases = []ConfigInput{
+	{
+		Name:             "serviceentry-workloadentry",
+		Services:         100,
+		Instances:        1000,
+		KubernetesClient: true,
+	},
+}
+
+func BenchmarkAddressFullGeneration(b *testing.B) {
+	runBenchmark(b, v3.AddressType, wdsCases)
+}
+
+func TestAddressFullGeneration(t *testing.T) {
+	testBenchmark(t, v3.AddressType, wdsCases)
+}
+
+var wdsIncrementalCases = func() []ConfigInput {
```

Comment:
Just trivia: why set the var to an immediately executing function?

Reply:
We are trying to compute a var one time and store it to a global var. Only way to do that is via a function call (or start it unitialized and set in `init()` but I think that is much uglier)

Reply:
Ah - couldn't tell this was global in the diff - thanks for explaining!